service-commands/scripts/init-repos.sh

```diff
@@ -32,18 +34,17 @@ npm install &
 # setup creator node
 cd $PROTOCOL_DIR/
 cd creator-node/
-npm install &
-npm install & #why does it not work without this?
+npm install
 
 # setup libs
 cd $PROTOCOL_DIR/
 cd libs/
-npm install &
+npm install
 
 # setup identity service
 cd $PROTOCOL_DIR/
 cd identity-service/
-npm install --dev &
+npm install --dev
```

Comment:
is removing this tested?

Reply:
yes, but somehow it still doesn't install everything the right way the first time through. looking into this.

Reply:
fixed and tested!